Source/WebCore/bindings/scripts/test/JS/JSTestIndexedSetterThrowingException.cpp

```diff
@@ -219,6 +219,9 @@ bool JSTestIndexedSetterThrowingException::put(JSCell* cell, JSGlobalObject* lex
     }
 
     throwScope.assertNoException();
+    if (!parseIndex(propertyName))
```

Comment:
If parseIndex() returned a valid index, we would have taken the branch at line 214 and returned early at line 218.
Therefore, it seems like this condition here should always be true.

Reply:
Fixed.